adaptors/backbone/context_adaptor.js

```diff
@@ -32,6 +60,12 @@ var lookupValue = function(view, name) {
   if (this.isModel(view) && view.has(name)) {
     value = view.get(name);
   } else if (view[name]) {
+    if (view.tungstenCollection && blockedCollectionProperties[name]) {
+      return null;
+    }
+    if (view.tungstenModel && blockedModelProperties[name]) {
+      return null;
```

Comment:
This prevents values present on Backbone prototypes from being accidentally accessed via template bubbling. Few if any should be validly accessed from a template and are only checked due to the upstream checks that Mustache performs. With the whitelist at the top of the file, it's easy to add values that are validly accessed, such as validationErrors. Returning null here lets the lookup function continue past this match rather than getting the value of Backbone.Model.prototype[name]. If the current model contains a matching property, the condition on line 60 would have caught and this isn't relevant.